app/helpers/sap_azure_plan_helper.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+# helper for rendering `terraform show` output as a plan
+module SapAzurePlanHelper
```

Comment:
I guess with this new PR: https://github.com/SUSE-Enceladus/blue-horizon/pull/176

This code could go in `CustomHelper`, right?

Reply:
exactly! I just merged that so i'm gonna sync the fork and rebase.